contracts/governance/Vesting/VestingLogic.sol

```diff
@@ -13,6 +13,10 @@ import "./VestingStorage.sol";
  * @title Vesting Logic contract.
  * @notice Staking, delegating and withdrawal functionality.
  * @dev Deployed by a VestingFactory contract.
+ *
+ * In the previous version, there is public functions that is removed:
+ * governanceWithdrawTokens(address _receiver)
+ * We removed this because we changed the flow for governance vesting withdrawal from Staking -> Vesting -> Staking, into directly frorm staking.
```

Comment:
typo
```suggestion
* We removed this because we changed the flow for governance vesting withdrawal from Staking -> Vesting -> Staking, into directly from staking.
```